hey bro , how did you fix this Lint and review content files / lint-and-review-docs (pull_request_target)  error ?

Logs from markdownlint:
yarn run v1.22.22
$ /home/runner/work/translated-content/translated-content/node_modules/.bin/markdownlint-cli2 --fix files/zh-cn/web/css/_doublecolon_after/index.md
markdownlint-cli2 v0.13.0 (markdownlint v0.34.0)
Finding: files/zh-cn/web/css/_doublecolon_after/index.md !node_modules !.git !.github !**/conflicting/** !**/orphaned/**
Linting: 1 file(s)
Summary: 0 error(s)
Done in 0.30s.

Logs from front-matter linter:
yarn run v1.22.22
$ node scripts/front-matter_linter.js --fix true --config-file /home/runner/work/translated-content/translated-content/front-matter-config.json /home/runner/work/translated-content/translated-content/files/zh-cn/web/css/_doublecolon_after/index.md
0 0
Done in 0.55s.

Please fix all the linting issues mentioned in above logs and in the review comments.
Error: Process completed with exit code 1.

--- a/files/zh-cn/web/css/_doublecolon_after/index.md
+++ b/files/zh-cn/web/css/_doublecolon_after/index.md
@@ -11,7 +11,7 @@ l10n:
 
 {{EmbedInteractiveExample("pages/tabbed/pseudo-element-after.html", "tabbed-standard")}}
 
-> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“起源元素”的直接子元素，因此不能应用于[*替换元素*](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement(“img”)}}），它们的内容被替换，不受当前文档样式的影响。
+> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“起源元素”的直接子元素，因此不能应用于[_替换元素_](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement(“img”)}}），它们的内容被替换，不受当前文档样式的影响。
 
 ## 语法
 
@@ -91,18 +91,17 @@ l10n:
 
 本例使用 `::after`，结合 [`attr()`](/zh-CN/docs/Web/CSS/attr) CSS 表达式和 `data-descr` [自定义数据属性](/zh-CN/docs/Web/HTML/Global_attributes/data-*)，创建工具提示。无需 JavaScript！
 
-我们还可以使用此技术为键盘用户提供支持，方法是添加一个值为 `0` 的 `tabindex` 使每个 `span` 都可在键盘上聚焦，并使用 CSS `:focus` 选择器。这说明了 
- `::before` 和 `::after` 可以多么灵活，不过要获得最方便的体验，以其他方式创建的语义披露部件（如使用 [detail 和 summary](/zh-CN/docs/Web/HTML/Element/details) 元素）可能更合适。
+我们还可以使用此技术为键盘用户提供支持，方法是添加一个值为 `0` 的 `tabindex` 使每个 `span` 都可在键盘上聚焦，并使用 CSS `:focus` 选择器。这说明了 `::before` 和 `::after` 可以多么灵活，不过要获得最方便的体验，以其他方式创建的语义披露部件（如使用 [detail 和 summary](/zh-CN/docs/Web/HTML/Element/details) 元素）可能更合适。
 
 #### HTML
 
 ```html
 <p>
-  这里我们有包含了一些<span tabindex="0" data-descr="鼠标悬停时出现的小弹出窗口">
+  这里我们有包含了一些<span
+    tabindex="0"
+    data-descr="鼠标悬停时出现的小弹出窗口">
     工具提示</span
-  >的<span tabindex="0" data-descr="文字和标点符号的集合">
-    文字
-  </span>。
+  >的<span tabindex="0" data-descr="文字和标点符号的集合">文字</span>。
 </p>
 ```
 

--- a/files/zh-cn/web/css/_doublecolon_after/index.md
+++ b/files/zh-cn/web/css/_doublecolon_after/index.md
@@ -11,7 +11,7 @@ l10n:
 
 {{EmbedInteractiveExample("pages/tabbed/pseudo-element-after.html", "tabbed-standard")}}
 
-> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“起源元素”的直接子元素，因此不能应用于[_替换元素_](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement(“img”)}}），它们的内容被替换，不受当前文档样式的影响。
+> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“起源元素”的直接子元素，因此不能应用于[_替换元素_](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement("img")}}），它们的内容被替换，不受当前文档样式的影响。
 
 ## 语法
 

--- a/files/zh-cn/web/css/_doublecolon_after/index.md
+++ b/files/zh-cn/web/css/_doublecolon_after/index.md
@@ -7,7 +7,7 @@ l10n:
 
 {{CSSRef}}
 
-在 CSS 中，**`::after`** 会创建一个[伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)，作为所选元素的最后一个子元素。它通常用于为具有 {{CSSxRef(“content”)}} 属性的元素添加修饰内容。默认情况下，它是行向布局的。
+在 CSS 中，**`::after`** 会创建一个[伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)，作为所选元素的最后一个子元素。它通常用于为具有 {{CSSxRef("content")}} 属性的元素添加修饰内容。默认情况下，它是行向布局的。
 
 {{EmbedInteractiveExample("pages/tabbed/pseudo-element-after.html", "tabbed-standard")}}
 

